_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+id: Custom Resource Definition
+name: Custom Resource Definition
+tags:
+- fundamental
+- operation
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
```

Comment:
This link mainly talks about TPRs. Please link to https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/.

Reply:
done 😃